iOS/Plugins/FlipperKitLayoutPlugin/FlipperKitLayoutPlugin/descriptors/SKViewDescriptor.mm

```diff
@@ -452,17 +450,21 @@ - (NSArray*)validChildrenForNode:(UIView*)node {
     @"Accessibility.accessibilityTraits.UIAccessibilityTraitCausesPageTurn": ^(NSNumber *value) {
       node.accessibilityTraits = AccessibilityTraitsToggle(node.accessibilityTraits, UIAccessibilityTraitCausesPageTurn, [value boolValue]);
     },
-    @"Accessibility.accessibilityTraits.UIAccessibilityTraitTabBar": ^(NSNumber *value) {
-      node.accessibilityTraits = AccessibilityTraitsToggle(node.accessibilityTraits, UIAccessibilityTraitTabBar, [value boolValue]);
-    },
     @"Accessibility.accessibilityViewIsModal": ^(NSNumber *value) {
       node.accessibilityViewIsModal = [value boolValue];
     },
     @"Accessibility.shouldGroupAccessibilityChildren": ^(NSNumber *value) {
       node.shouldGroupAccessibilityChildren = [value boolValue];
     },
-}
-;
+  };
+  if (@available(iOS 10.0, *)) {
+    NSMutableDictionary<NSString*, SKNodeUpdateData> *latestDataMutations = [dataMutations mutableCopy];
+    latestDataMutations[@"Accessibility.accessibilityTraits.UIAccessibilityTraitTabBar"] = ^(NSNumber *value) {
+      node.accessibilityTraits = AccessibilityTraitsToggle(node.accessibilityTraits, UIAccessibilityTraitTabBar, [value boolValue]);
+    };
+    dataMutations = [latestDataMutations copy];
```

Comment:
Happy to merge it like this but I got the feedback internally, that it would be fine to skip this as we have no further references to the mutable copy and could cast it.

Reply:
Works for me; I’ll make the change 👍

Reply:
Done ✅